include/fem-dem/vans_assemblers.h

```diff
@@ -142,25 +143,24 @@ class GLSVansAssemblerBDF : public NavierStokesAssemblerBase<dim>
 };
 
 /**
- * @brief Class that assembles the particle_fluid interactions (PFI) for the
- * VANS equations such as the drag force.
+ * @brief Class that assembles the drag force using DiFelice model for the
+ * VANS equations.
```

Comment:
Would be a good idea to add the equation for the Di Felice drag model here.

Reply:
done